packages/origin-backend-client-mocks/src/DeviceClientMock.ts

```diff
@@ -77,4 +70,17 @@ export class DeviceClientMock implements IDeviceClient {
 
         this.storage.set(id, device);
     }
+
+    public async getSupplyBy(facilityName: string, status: number): Promise<IDeviceWithRelationsIds[]> {
+        return [...this.storage.values()];
+    }
+
+    public async delete(id: number): Promise<void> {
+        
+    }
```

Comment:
There were some `throw()` methods here. What's the reason for removing them and this being empty now?

Reply:
do i need put back as though that are reason of test failed